src/scss/components/switch/_index.scss

```diff
@@ -0,0 +1,178 @@
+@import "./config";
+
+hx-switch-control {
+  -ms-grid-column-align: start;
+  -ms-grid-columns: auto 1fr;
+  -ms-grid-rows: auto auto;
+  display: -ms-grid;
+  display: grid;
+  grid-template-areas:
+    "ctrl lbl"
+    ".    help";
+  grid-template-columns: auto 1fr;
+  grid-template-rows: auto auto;
+  justify-items: start;
+  grid-area: facade;
+  margin-top: 0.25rem; // FIXME: fix with proper inline styling
+  position: relative;
+  width: 4.25rem;
+  height: 2rem;
+
+  > input {
+    opacity: 0;
+    width: 0;
+    height: 0;
+  }
+}
+
+hx-switch-control > input[type="checkbox"] ~ label {
+  > hx-switch {
+    background-color: #ffffff;
+    border: 0.125rem solid #0c7c84;
```

Comment:
Please update to use our Helix Color Palette colors/variables:

https://github.com/HelixDesignSystem/helix-ui/blob/master/src/scss/_vars.scss

Reply:
Changes have been done, please review.